.gitignore

```diff
@@ -5,10 +5,14 @@ coverage
 /dist
 /test/regressions/screenshots/output
 /test/selenium-output
+.eslintcache
 
 # OS files
 .DS_STORE
 
 # npm files
 node_modules
 *.log
+
+# Editor files
+.idea
```

Comment:
Should most likely be stored into you global `.gitignore` file, but that's not a big issue.